app/services/auto-approval/checks/is-no-previous-pending-claim.js

```diff
@@ -0,0 +1,17 @@
+const statusEnum = require('../../../constants/status-enum')
+const AutoApprovalCheckResult = require('../../domain/auto-approval-check-result')
+
+const CHECK_NAME = 'isNoPreviousPendingClaim'
+const FAILURE_MESSAGE = 'There was a previous pending claim for this claimant'
+
+module.exports = function (autoApprovalData) {
+  for (var i = 0; i < autoApprovalData.previousClaims.length; i++) {
+    var claim = autoApprovalData.previousClaims[i]
+
+    if (claim.Status === statusEnum.PENDING) {
```

Comment:
pending this rule does not mean just status PENDING, but a claim not approved or rejected